sktime/datasets/_data_io.py

```diff
@@ -135,7 +136,38 @@ def _list_available_datasets(extract_path):
 
 
 def _load_dataset(name, split, return_X_y, return_type=None, extract_path=None):
```

Comment:
maybe this could become public? It is used in testing elsewhere I think

Reply:
well, that would not be a refactor/integration but a separate change.

Makes sense to me, but perhaps better to do this in a separate PR.